mdbenchmark/generate.py

```diff
@@ -71,6 +71,11 @@ def generate(name, gpu, module, host, min_nodes, max_nodes, time, list_hosts):
         print_possible_hosts()
         return
 
+    if not name:
```

Comment:
isn't there a click option to print this automatically?

Reply:
Only if we define it as an argument, then we could use `click.File` inside of that definition.